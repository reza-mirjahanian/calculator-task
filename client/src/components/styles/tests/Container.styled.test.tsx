import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import {Container} from "../Container.styled";



test('Container should works', () => {
    const tree = renderer.create(<Container >Reza</Container>).toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree).toHaveStyleRule('padding', '0 20px');
    expect(tree).toHaveStyleRule('margin', '0 auto');
    expect(tree).toHaveStyleRule('min-height', '100vh');
})
