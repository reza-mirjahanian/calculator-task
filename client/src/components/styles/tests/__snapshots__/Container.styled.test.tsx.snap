// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should works 1`] = `
.c0 {
  width: 1000px;
  max-width: 100%;
  padding: 0 20px;
  margin: 0 auto;
  min-height: 100vh;
}

<div
  className="c0"
>
  Reza
</div>
`;
